package.json

```diff
@@ -35,9 +48,14 @@
     "sinon": "^7.2.7",
     "sinon-as-promised": "^4.0.3",
     "stylelint": "^9.4.0",
-    "stylelint-use-logical": "^1.1.0"
+    "stylelint-use-logical": "^1.1.0",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.3.0"
   },
   "engines": {
     "node": ">=10.12"
+  },
+  "scripts": {
+    "test": "jest"
```

Comment:
This command should test everything: widen to include sass & js testing, & update in the future as necessary.